addon/components/prismic/element.js

```diff
@@ -32,4 +32,12 @@ export default class PrismicElementComponent extends Component {
   get target() {
     return this.args.node.element.data.value?.target || '_blank';
   }
+
+  get isCustom() {
+    return !!this.componentName;
```

Comment:
May I suggest you to use the explicit syntax of boolean conversion, for readability purpose?

How do you feel about the following?
```suggestion
    return Boolean(this.componentName);
```

Reply:
Aah, I could of just pressed the 'Commit suggestion' button :), but instead force pushed update :(. Have updated.

Reply:
Awesome, thanks for this one